packages/office-addin-dev-settings/src/commands.ts

```diff
@@ -165,6 +165,24 @@ export async function enableRuntimeLogging(path?: string) {
   }
 }
 
+export async function getWebView() {
+  try {
+    const webview = await devSettings.getWebView();
+    webview ? console.log("The webview has been set to " + webview + ".") : 
+    console.log("A specific webview has not been selected. Edge is used as a default.");
+  } catch (err) {
+    logErrorMessage(err);
+  }
+}
+
+export async function setWebView(webview?: string) {
```

Comment:
Can't we just check if webview is set in start.ts and then only call setWebView if it is, and then **not** make webview and optional param?